Resources/Audio/Voice/doll/Copyright © Epidemic Sound

```diff
@@ -0,0 +1 @@
+https://www.epidemicsound.com/music/featured/
```

Comment:
The standard way to attribute copyrights these days is to use the RGA format.

There's a good example in `Resources/Audio/Items/Geiger/attributions.yml` and documentation is listed here: https://github.com/space-wizards/RobustToolboxSpecifications/blob/0bed2921812703e1a32c9bc98fc4d7cba4a00289/RobustGenericAttribution/README.md